scripts/check-mypy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# This list will be expanded as violations are addressed.
+MYPY_SRC_FILES=(
+	src/instructlab/log.py
+)
```

Comment:
It seems wrong to have this limited include list in here. I realize that the current code base will fail though. (sigh)

Reply:
That's a good idea. I believe `mypy` support `--exclude` as well as config through `pyproject.toml`